RetailCoder.VBE/UI/Command/AddTestModuleCommand.cs

```diff
@@ -85,6 +98,16 @@ private string GetTestModule(IUnitTestSettings settings)
             return formattedModuleTemplate;
         }
 
+        private string GetBinding(bool isEarlyBinding, string theType)
```

Comment:
I don't like the `the` prefix; `theType` could just as well be `type`.

Given above `InstantiateEarlyBoundFormat` / `InstantiateLateBoundFormat` naming alternative, this method would seem more natural like this:

    private string FormatInstantiationFor(string type, BindingMode binding)

And called like `var assertDeclaration = $"Set Assert = {FormatInstantiationFor(assertType, settings.BindingMode)}";`